### ISSUE TITLE
Block Sizes for Exemplum

### ISSUE THREAD

**Issue by otsfan203** [exemplumscratchmod/exemplum-blocks#4]
While we already know how to toggle block sizes, it's necessary to discuss **what** sizes those are going to be.

@Lightningfoot could you help?

Right now I'm thinking that it should be exactly at the half point of [Sheeptester's Horizontal Playground](https://sheeptester.github.io/scratch-blocks/playgrounds/tests/vertical_playground.html) and the Scratch 3.0 blocks.

What does everyone else think?

**Comment by Lightningfoot** [comment]
I think that that's the correct size

**Comment by otsfan203** [comment]
@Lightningfoot I think it's a good size. However, the original purpose of the bigger blocks were for mobile and tablet. While the blocks shouldn't be oversized, a little bit bigger would be fine? I'm not sure. I'll make a Strawpoll for it, and we can all vote for it.

**Comment by otsfan203** [comment]
@Lightningfoot I finished. https://strawpoll.com/pxg3d98x

**Comment by Lightningfoot** [comment]
Yeah, I voted for a bit bigger, I think that it should be a little bigger than to 2.0 blocks but not quite as big as the 3.0 blocks.

**Comment by otsfan203** [comment]
@Lightningfoot Yep. Since using JavaScript and HTML5 Player allows for mobile support, the blocks need to be a bit bigger, but it doesn't make sense to make sense to make them obnoxiously large.
